bluewin.ch

```diff
@@ -1,41 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
-  <emailProvider id="bluewin.ch">
+  <emailProvider id="Swisscom">
     <domain>bluewin.ch</domain>
-    <!-- See also bluemail.ch -->
-    <displayName>bluewin.ch</displayName>
-    <displayShortName>bluewin.ch</displayShortName>
+    <domain>bluemail.ch</domain>
+    <displayName>Bluewin Mail</displayName>
+    <displayShortName>Bluewin</displayShortName>
     <incomingServer type="imap">
       <hostname>imaps.bluewin.ch</hostname>
       <port>993</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
       <hostname>pop3s.bluewin.ch</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <outgoingServer type="smtp">
       <hostname>smtpauths.bluewin.ch</hostname>
       <port>465</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-encrypted</authentication>
+      <username>%EMAILADDRESS%</username>
+      <authentication>password-cleartext</authentication>
```

Comment:
I know this is SSL, but hashed passwords are always better than plaintext passwords.

Please try whether encrypted logins work. Try all of IMAP, POP3 and SMTP. If they work, then we want to use them, not plaintext. If they work only for SMTP, we keep it there.

Reply:
The encrypted password types that are currently still available are planned to be phased out this is in preparation of this step.
The auth mechanisms available don't allow optimal storage of credentials at rest and will thus be removed.

Reply:
Are you working for Swisscom?

Please note that these configurations will only affect new users. They will not update old users. Thus, you cannot remove existing authentication mechanisms, unless you want to break all existing users.

Reply:
I am, and we are aware of that, the necessary steps to communicate these changes will be taken.